Doctrine: Migration: added functionality for ignore column from entity by annotation (#2368)
shopsys/shopsys: docs/extensibility/entity-extension.md

    To see how it works in practice, check out `\Tests\App\Functional\EntityExtension\EntityExtensionTest` that tests end-to-end extensibility of `Product`, `Category` and `OrderItem`.

### How can I remove property from parent entity?

> There is `#[RemoveColumn]` that you can use to remove a column from the parent entity.

Reviewer: ```suggestion
You can use the `#[RemoveColumn]` attribute to remove a column from the parent entity.
```
